content/v2.0/application/settings.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Settings
+order: 10
+---
+
+Hanami supports application settings through a `Settings` class, defined in `config/settings.rb`.
+
+```ruby
+# config/settings.rb
+
+# frozen_string_literal: true
+
+module Bookshelf
+  class Settings < Hanami::Settings
+    # Define your app settings here, for example:
+    #
+    # setting :my_flag, default: false, constructor: Types::Params::Bool
+  end
+end
+```
+
+Using this class, you can define what settings exist within your application, what types and defaults they have, and whether or not they are required for your application to boot.
+
+Each setting is read from an environment variable matching its name. For example, with the following configuration, the Redis URL and Sentry DSN are sourced from the `REDIS_URL` and `SENTRY_DSN` environment variables.
```

Comment:
```suggestion
Each setting is read from an environment variable matching its name. For example, with the following settings, the Redis URL and Sentry DSN are sourced from the `REDIS_URL` and `SENTRY_DSN` environment variables.
```

For the same reasons I outlined in my first comment, let's stick to "settings" over "configuration" here whenever we can.

Reply:
Done, thanks.